db/db_impl/db_impl.h

```diff
@@ -1763,7 +1763,7 @@ class DBImpl : public DB {
   // to have acquired the SuperVersion and pass in a snapshot sequence number
   // in order to construct the LookupKeys. The start_key and num_keys specify
   // the range of keys in the sorted_keys vector for a single column family.
-  void MultiGetImpl(
+  Status MultiGetImpl(
```

Comment:
Nit: I think the return value of `MultiGetImpl` worth a sentence. Now that it returns a `Status` now, it may be clearer to explicitly say the return value is either OK or timedout, while the result of individual key lookups are stored in `sorted_keys`.